web/api/Makefile

```diff
@@ -1,14 +1,60 @@
-# Check if certain environmental variables are set properly.
-include Makefile.check
+THRIFT_OPTS = -r -o /data
+TARGET_PY = --gen py
+TARGET_JS = --gen js:jquery --gen js:node
 
-# Execute the build of each API subfolder.
-SUBDIRS := $(wildcard */.)
+current_dir = $(shell pwd)
+uid = $(shell id -u)
+guid = $(shell id -g)
 
-default: all
+API_DIR := "$(current_dir)"
 
-all: $(SUBDIRS)
+PY_API_DIR = "./py/codechecker_api/codechecker_api"
+PY_API_SHARED_DIR = "./py/codechecker_api_shared/codechecker_api_shared"
+JS_API_DIR = "./js/codechecker_api_js/lib/codechecker_api_js"
```

Comment:
You can put the generated javascript files inside the lib directory so there is no the to create another directory inside the lib directory.